packages/headless/src/features/case-assist/case-assist-analytics-actions.ts

```diff
@@ -0,0 +1,90 @@
+import {makeCaseAssistAnalyticsAction} from '../analytics/analytics-utils';
+import {
+  selectCase,
+  selectCaseClassification,
+  selectDocumentSuggestion,
+} from './case-assist-analytics-selectors';
+
+export const logCaseStart = makeCaseAssistAnalyticsAction(
```

Comment:
Do we want the case assist engine to log this action when contructed? I don't think it would fit into the lifecycle of any of our other controllers so if not then it would be on the implementer to dispatch it.

Reply:
That is indeed a good point. I'll look into it.

Reply:
The case start action allows the implementer to pass in the case information. However, it would not be possible to pass the case information if the action is logged upon engine creation.